stable/instana-agent/templates/daemonset.yaml

```diff
@@ -110,6 +110,10 @@ spec:
             - name: configuration
               subPath: configuration.yaml
               mountPath: /root/configuration.yaml
+            {{- if .Values.agent.host.repository }}
+            - name: repo
+              mountPath: /opt/instana/agent/data/repo
```

Comment:
Should this be `{{ .Values.agent.host.repository }}` ?

Reply:
The `mountPath`? Nope - it is hard coded to point to the agent repo location because we're mounting into that location.

Reply:
So what would the value of `.Values.agent.host.repository` be?

Reply:
Clarified in person